kitchen/src/kitchen/core.clj

```diff
@@ -448,9 +448,14 @@
               (let [num-chars-str (subs in-data (count "chars-"))
                     num-chars-int (Integer/parseInt num-chars-str)
                     chars (map char (range 65 91))
-                    string-data (->> (repeatedly #(rand-nth chars))
-                                     (take num-chars-int)
-                                     (reduce str))]
+                    seed-string (->> (repeatedly #(rand-nth chars))
```

Comment:
Can we factor this string generation out into a separate function?   It could certainly be faster but imagine this is fast enough.